cmd/horcrux/cmd/cosigner.go

```diff
@@ -30,12 +30,14 @@ var cosignerCmd = &cobra.Command{
 
 func StartCosignerCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "start [single-signer]",
+		Use:   "start [single-signer] [rpc-timeout]",
```

Comment:
because both of these are flags, I don't think we need to put them as `args` here